stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/HttpConnectionManager.java

```diff
@@ -197,6 +195,47 @@ public String getHttpAccessLoggerConfig() {
         return ConfigRegistry.getInstance().getConfiguration(HTTP_ACCESS_LOG, LOG_TO);
     }
 
+    private TransportsConfiguration buildDefaultTransportConfig() {
+        TransportsConfiguration transportsConfiguration = new TransportsConfiguration();
+        ListenerConfiguration httpListener = new ListenerConfiguration("default", "0.0.0.0", 9090);
```

Comment:
Can we make these constants? May be they are used only here, still it would make maintenance easier.